### ISSUE TITLE
Can this be used with react-native?

### ISSUE THREAD

**Issue by MadalinaCi** [DJanoskova/vuex-but-for-react#4]
Hi! This is an awesome package! I would love to use this in my react-native app, but I am not sure if it's supported or how can I change it to use it with react-native projects. Any help is highly appreciated.

**Comment by zsorzs7** [comment]
i am planning to use it, too. could you use it?

**Comment by visitsb** [comment]
@MadalinaCi , @zsorzs7 Yes 👍 , this fork [@visitsb/vuex](https://visitsb.github.io/vuex/) allows you to use Vuex _as-is_ inside React, ReactJS and React Native applications. 

@DJanoskova Thank for your 🥇 excellent inspiration and work. I digged around your source, and was trying to make your library as close to Vuex as possible. But in the end found a [much easier way](https://github.com/vuejs/vuex/pull/2149) to use Vuex exactly the same way but _outside_ of Vue applications. There is an example of how to use Vuex + React Native [here](https://visitsb.github.io/vuex/guide/). Would definitely 👍  love to hear from you about this [library](https://visitsb.github.io/vuex/).
